api/opam.ts

```diff
@@ -0,0 +1,77 @@
+import cheerio from 'cheerio'
+import got from '../libs/got'
+import { millify, version, versionColor } from '../libs/utils'
+import { createBadgenHandler, PathArgs } from '../libs/create-badgen-handler'
+
+const OPAM_REPO_URL = 'https://opam.ocaml.org/packages/'
+
+type PackageInfo = {
+  version: string,
+  license: string,
+  downloads: number
+}
+
+export default createBadgenHandler({
+  title: 'opam',
```

Comment:
We can give it a full title for better seo and search matching

```suggestion
  title: 'OCaml Package Manager',
```

Reply:
Thx for fixing that (and missing EOF newline) for me :+1: